pom.xml

```diff
@@ -18,16 +19,16 @@
     </description>
 
     <scm>
-        <connection>scm:git:git@github.com:jenkinsci/jsch-plugin.git</connection>
-        <developerConnection>scm:git:git@github.com:jenkinsci/jsch-plugin.git</developerConnection>
-        <url>https://github.com/jenkinsci/jsch-plugin/</url>
+        <connection>scm:git:git://github.com/jenkinsci/${project.artifactId}-plugin.git</connection>
```

Comment:
Note change of URL format. Bringing this into line with other plugins. I suspect this is at the root of https://github.com/jenkinsci/bom/pull/36 PCT failures.